src/new.rs

```diff
@@ -0,0 +1,125 @@
+#![allow(non_upper_case_globals)]
+
+const cobalt_yml: &'static [u8] = b"name: cobalt blog
+source: \".\"
+dest: \"build\"
+ignore:
+  - ./.git/*
+  - ./build/*
+";
+
+const default_liquid: &'static [u8] = b"<!DOCTYPE html>
+<html>
+    <head>
+      <meta charset=\"utf-8\">
+    	<meta http-equiv=\"X-UA-Compatible\" content=\"IE=edge\">
+    	<meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">
+
+        {% if is_post %}
+          <title>{{ title }}</title>
+        {% else %}
+       	  <title>Cobalt.rs Blog</title>
+        {% endif %}
+
+	    <!-- Latest compiled and minified CSS -->
+     	<link rel=\"stylesheet\" href=\"https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css\" integrity=\"sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7\" crossorigin=\"anonymous\">
+      	<!-- Optional theme -->
+     	<!--<link rel=\"stylesheet\" href=\"https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap-theme.min.css\" integrity=\"sha384-fLW2N01lMqjakBkx3l/M9EahuwpSfeNvV63J5ezn3uZzapT0u7EYsXMjQV+0En5r\" crossorigin=\"anonymous\">-->
+
+      <script src=\"https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js\"></script>
```

Comment:
mmh let's not include all these resources for now. let's instead make a follow up bug that thinks about a good set of default files/resources to include